corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js

```diff
@@ -171,6 +171,163 @@ hqDefine("hqwebapp/js/knockout_bindings.ko", ['jquery', 'knockout', 'jquery-ui/u
         },
     };
 
+    ko.bindingHandlers.multi_sortable = {
```

Comment:
How about `multirow_sortable`? `multi_sortable` sounds like multisort, which I think is more commonly understood to mean sorting by multiple criteria.